jobs/migrations/0008_auto_20201006_AddStateData.py

```diff
@@ -0,0 +1,126 @@
+# Generated by Django 3.1 on 2020-10-07 01:42
+from __future__ import unicode_literals
+from django.db import migrations, models
+
+def add_states(apps, schema_editor):
```

Comment:
Do these match the `area_title` values in the BLS OES dataset? I think they need to match so we can look things up. Do we also need a national code? I'm not sure what the BLS dataset uses for that.

Reply:
Added issue #122 to handle this. It was not in the original scope but I agree we should be working towards it.